### PR TITLE
Update Gitea Operator to v2

### PR DESCRIPTION
**Pull request by cfahrni**
Update Gitea Operator to v2

The previously referenced Gitea Operator was moved to another repo and was rewritten/updated.

https://github.com/rhpds/gitea-operator

### CODE REVIEW

**Review by bliemli — COMMENTED** [review]
Thank you!

File: content/en/docs/05/01_setup.md [hunk 1]
```diff
@@ -14,26 +14,18 @@ It allows us to easily host a Git repository which we can use as our single sour
 
 Unfortunately, the default Operator catalog sources don't contain a Gitea operator, but let's change that.
 
-Add the following catalog source to your cluster:
+Create the Gitea Operator by applying the YAML files from the Github repository
 
 ```bash
-oc apply -f https://raw.githubusercontent.com/redhat-gpte-devopsautomation/gitea-operator/master/catalog_source.yaml
+oc apply -k https://github.com/rhpds/gitea-operator/OLMDeploy
```
[the remark this answers is not in the document]
Comment (cfahrni) [reply content/en/docs/05/01_setup.md:20]: The command is [used in the documentation](https://github.com/rhpds/gitea-operator/#installing-the-operator) and worked in my labs.
```
> oc apply -k https://github.com/rhpds/gitea-operator/OLMDeploy
namespace/gitea-operator unchanged
operatorgroup.operators.coreos.com/gitea-operator configured
catalogsource.operators.coreos.com/redhat-rhpds-gitea unchanged
subscription.operators.coreos.com/gitea-operator unchanged
```

[the remark this answers is not in the document]
Comment (bliemli) [reply content/en/docs/05/01_setup.md:20]: Interesting, GitHub seems to respond differently based on the user agent.

[the remark this answers is not in the document]
Comment (cfahrni) [reply content/en/docs/05/01_setup.md:20]: This made me curious too ;) 

`oc apply -k` seems to clone the git repo and run a `kustomize build` inside the directory according to [their documentation](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#remote-directories).

**Pull request merged**